### PR TITLE
Update maintainers.MD with steps to publish package

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -26,10 +48,29 @@ Minio .NET Library uses nuget for its dependency management https://nuget.org/
...
+> nuget pack Minio.nuspec
+> nuget pack Minio.core.nuspec
 ... package built ...
 ```
 
 #### Go to nuget.org
 
-Sign into nuget.org to [upload a package through browser](https://www.nuget.org/users/account/LogOn?ReturnUrl=%2Fpackages%2Fupload).
\ No newline at end of file
+Sign into nuget.org to [upload a package through browser](https://www.nuget.org/users/account/LogOn?ReturnUrl=%2Fpackages%2Fupload).
```
Comment (harshavardhana) [inline MAINTAINERS.md]: We can say s/Sign/Login

File: MAINTAINERS.md [hunk 2]
```diff
@@ -26,10 +48,29 @@ Minio .NET Library uses nuget for its dependency management https://nuget.org/
...
+Upload both the packages generated by the previous step.
+
+#### Tag
+
```
Comment (harshavardhana) [inline MAINTAINERS.md]: Remove the line here..

File: MAINTAINERS.md [hunk 3]
```diff
@@ -26,10 +48,29 @@ Minio .NET Library uses nuget for its dependency management https://nuget.org/
...
+Tag and sign your release commit, additionally this step requires you to have access to Minio's trusted private key.
+
+$ export GNUPGHOME=/media/${USER}/minio/trusted
+$ git tag -s 1.0.2
+$ git push
+$ git push --tags
+
+#### Announce
+
```
Comment (harshavardhana) [inline MAINTAINERS.md]: Remove the above line..

File: MAINTAINERS.md [hunk 4]
```diff
@@ -15,6 +15,28 @@ Fork [minio-dotnet upstream](https://github.com/minio/minio-dotnet/fork) source
...
+ <Version>1.0.2</Version>
+Update Minio.Net452/Properties/AssemblyInfo.cs with the next AssemblyVersion and AssemblyFileVersion
+[assembly: AssemblyVersion("1.0.2.0")]
+[assembly: AssemblyFileVersion("1.0.2.0")]
+Update the <version> tag in Minio.nuspec and Minio.core.nuspec
+
+#### Build
+```powershell
+> dotnet msbuild /p:Configuration=Release
+
```
Comment (harshavardhana) [inline MAINTAINERS.md]: Remove the line..

**Comment by poornas** [comment]
done.

**Pull request merged**